src/loader/bgw_launcher.c

```diff
@@ -48,12 +48,32 @@ typedef enum AckResult
 	ACK_SUCCESS,
 } AckResult;
 
+/* See state machine in README.md */
+typedef enum SchedulerState
+{
+	/* Scheduler should be started but has not been allocated or started */
+	ENABLED = 0,
+	/* The scheduler has been allocated a spot in timescaleDB's worker counter */
+	ALLOCATED,
+	/* Scheduler has been started */
+	STARTED,
+	/* Scheduler is stopped and should not be started automatically. START and RESTART messages can re-enable the scheduler. */
+	DISABLED
+} SchedulerState;
+
 #ifdef TS_DEBUG
 #define BGW_LAUNCHER_RESTART_TIME 0
 #else
 #define BGW_LAUNCHER_RESTART_TIME 60
 #endif
 
+#ifdef TS_DEBUG
+#define BGW_LAUNCHER_POLL_TIME 10L
+#else
+#define BGW_LAUNCHER_POLL_TIME 1000L
```

Comment:
Do we have an idea what a reasonable poll rate is and why 1s is good? Ideally, we'd try to sleep as much as possible.

Reply:
changing to 60s